### DISCUSSION TITLE
Can't build the project on Heroku

### DISCUSSION THREAD

**Discussion by 3adel-bassiony** [adonisjs/core#3861] [Help] [answered]
Hello
Every time I try to deploy on Heroku I got this error, and I tried to build it on the local and it's built successfully without any issues, I don't know why it can't find the `model: () => import('App/Models/User')` but I build many apps before using Heroku and adonis and it built successfully!

Here is the error:
<img width="1218" alt="Screenshot 2022-08-12 at 7 02 57 PM" src="https://user-images.githubusercontent.com/81669622/184408104-67a003d7-14e2-4f6c-9b91-dbeecd991028.png">

**Comment by Barbapapazes** [comment]
Hello,


Can you provide a reproduction repository?
Thanks

**Reply by 3adel-bassiony** [reply]
I found the issue, When I setup the authentication and run this command:
`node ace configure @adonisjs/auth`
it asked me to enter the model name and I entered `user` not `User` so it created the model called user, and later I thought it's an issue so I renamed it to User but the git didn't notice the case sensitive change so on my local is `User` but on GitHub is `user`

I fixed it by renaming it to the correct one `User`

I think when setup the auth it should be following the naming convention even if the user entered it with small case, something like when creating a new model using this command line:
`node ace make:model user` => It will create a new model called User not user

**Comment by 3adel-bassiony** [comment, marked as answer]
I found the issue, When I set up the authentication and run this command:
node ace configure @adonisjs/auth
it asked me to enter the model name and I entered user, not User so it created the model called user, and later I thought it was an issue so I renamed it to User but the git didn't notice the case-sensitive change so on my local is User but on GitHub is user

I fixed it by renaming it to the correct one User

I think when setup the auth it should be following the naming convention even if the user entered it with a small case, something like when creating a new model using this command line:
node ace make:model user => It will create a new model called User not user
